metalearn/metafeatures/metafeatures.py

```diff
@@ -112,6 +110,16 @@ def compute(
             will be run to completion. Otherwise, execution will halt after
             approximately timeout seconds. Any metafeatures that have not been
             computed will be labeled 'TIMEOUT'.
+        groups: list, default None. Must consist only of values enumerated in
+            constants.MetafeatureGroup. The metafeature groups to be computed.
+            Values listed in metafeature_ids or exclude take precedence over
+            groups and exclude_groups (e.g., if 'landmarking' is in the list
+            exclude_groups but 'NaiveBayesErrRate' is in metafeature_ids,
+            NaiveBayesErrRate will be computed while all other landmarking
+            metafeatures will be excluded).
+        exclude_groups: list, default None. Must consist only of values enumerated in
+            constants.MetafeatureGroup. The metafeature groups to exclude from
+            computation. Must be None if groups is not None.
```

Comment:
Let's make it explicit that `groups` and `exclude_groups` are mutually exclusive. At least one has to be `None` and there is no precedence of one over the other. Add that to the validation code, if it's not already there. Make sure the logic using these values reflects that. Add a unit test for this explicitly, if its not there already.